### PR TITLE
🔥 feat: expiration per session

### PR DESCRIPTION
**Pull request by mirusky**
Please provide enough information so that others can review your pull request:

Added a sess.ExpiresIn() method who set a custom expiration for a session.

Explain the details for making this change. What existing problem does the pull request solve?

Related to #1298 , for short, sess.ExpiresIn() use a custom session expiration insted of default value.

### CODE REVIEW

**Review by hi019 — CHANGES_REQUESTED** [review]
Thanks for the PR. Did a small review from my phone, I'll take a closer look later today

File: middleware/session/session.go [hunk 1]
```diff
@@ -169,6 +176,11 @@ func (s *Session) Keys() []string {
 	return s.data.Keys()
 }
 
+// ExpiresIn set a specific expiration for this session
+func (s *Session) ExpiresIn(exp time.Duration) {
```
[the remark this answers is not in the document]
Comment (mlukasik-dev) [reply middleware/session/session.go:180]: And I'd better change comment to:  
`// ExpiresIn sets a specific expiration for this session` (grammatical mistake)

[the remark this answers is not in the document]
Comment (tomaswarynyca) [reply middleware/session/session.go:180]: @mirusky  could you generate the commit with the modifications?

**Comment by ReneWerner87** [comment]
@mirusky can you look over the review notes again and adjust accordingly so i can merge at some point

**Comment by mirusky** [comment]
Hey @ReneWerner87 @tomaswarynyca I did the changes, sorry for delay I was too busy.

**Review by ReneWerner87 — APPROVED** [review]

**Pull request merged**